IndicoV2.Abstractions/IndicoV2.Abstractions.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nothing is  being changed here

Reply:
it's just BOM :/ I'll check why there's this problem.